charts/cluster-scanner/templates/deployment.yaml

```diff
@@ -0,0 +1,398 @@
+          - name: NATS_JS_CONSUMER_PULL
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: rsi_js_consumer_pull
+          - name: NATS_JS_CONSUMER_SUBJECT
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: rsi_js_consumer_subject
+          - name: NATS_JS_CONSUMER_SUBJECT_PREFIX
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: rsi_js_consumer_subject_prefix
+          - name: NATS_JS_CONSUMER_MAX_IN_FLIGHT
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: rsi_js_consumer_max_in_flight
+          - name: NATS_JS_CONSUMER_ACK_WAIT
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: rsi_js_consumer_ack_wait
+          - name: NATS_JS_CONSUMER_MAX_DELIVER
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: rsi_js_consumer_max_deliver
+          - name: NATS_JS_PRODUCER_SUBJECT_PREFIX
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: rsi_js_producer_subject_prefix
+          - name: NATS_JS_SERVER_USERNAME
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: js_server_user
+          - name: SYSDIG_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: sysdig_access_key
+          - name: NATS_JS_SERVER_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "cluster-scanner.fullname" . }}
+                key: js_server_password
+          livenessProbe:
+            httpGet:
+              path: /probes/liveness
+              port: probes
+          readinessProbe:
+            httpGet:
+              path: /probes/readiness
+              port: probes
+      {{- with .Values.rsi }}
+          resources:
+            {{- toYaml .resources | nindent 12 }}
+      {{- end }}
+        - name: ise
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+      {{- with .Values.ise }}
+          image: "{{ .image.repository }}:{{ .image.tag }}"
+          imagePullPolicy: {{ .image.pullPolicy }}
+          ports:
+            - name: metrics
+              containerPort: {{ .ports.metrics }}
+              protocol: TCP
+            - name: probes
+              containerPort: {{ .ports.probes }}
+              protocol: TCP
```

Comment:
Unless they need to be accessed from other tools, probes port don't need to be exposed